.github/workflows/unit.yaml

```diff
@@ -0,0 +1,28 @@
+name: Unit tests
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6']
```

Comment:
```suggestion
        python-version: ['3.6', '3.7', '3.8', '3.9']
```

Reply:
Maybe 3.6 and 3.9, like for lint workflow?